ls1intum/Artemis
# `General`: Fix rounding issue in course statistics

### Checklist
#### General


#### Client

### Motivation and Context


Fix sentry issue

### Description

One method call did not include the course object for determining the accuracy of rounding in the course statistics. So sentry got notified and the default value of one decimal after the point was used.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Exercise with a finished participation

1. Log in to Artemis
2. Set the accuracy of scores to at least two in the course settings
3. Go to the course statistics
4. Check the tooltip of the overlay in the bar charts

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


Incorrectly rounding points to one decimal:
<img width="274" alt="Bildschirmfoto 2022-07-15 um 17 44 55" src="https://user-images.githubusercontent.com/38322605/179259198-0164d741-fa44-45d0-8163-49492490ec00.png">
Correctly rounding to two decimals as specified in the course settings:
<img width="274" alt="Bildschirmfoto 2022-07-15 um 17 42 54" src="https://user-images.githubusercontent.com/38322605/179259195-d67ad7db-5a20-44b4-afc3-fd4d937ac33b.png">
